api/src/test/resources/logback.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
```

Comment:
I noticed that this puts a bunch of stuff into the terminal when `sbt test` runs.  Can we have it go into a file instead, so there's not as much clutter in with the test output?  (Right now the output actually seems to come from logback reporting its own configuration.)

Also, it's warning that it finds `logback.xml` in two places.

I'm seeing similar output from slf4j, too.